src/main/java/com/test/contract/ui/MainController.java

```diff
@@ -84,12 +87,19 @@ public void init() {
      */
     @FXML
     public void addContact() {
-        Contact contact = new Contact(txtName.getText(), txtPhone.getText(), txtEmail.getText(), txtAge.getText());
+        Contact contact = new Contact(txtName.getText(), txtPhone.getText(), txtEmail.getText(), checkVariable(txtAge.getText()));
         contactService.save(contact);
         data.add(contact);
 
         txtName.setText("");
         txtPhone.setText("");
         txtEmail.setText("");
+        txtAge.setText("");
+    }
+
+    // Method for String variables verification
```

Comment:
You can use tern operator if: 

return str.matches("(?!0)[0-9]*") ? str : "";

Yours variant is not wrong, but this one [return str.matches("(?!0)[0-9]*") ? str : "";] looks more pretty))))